pkgs/tools/security/seclists/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+}:
+
+stdenv.mkDerivation rec {
+  pname = "seclists";
+  version = "2023.3";
+
+  src = fetchFromGitLab {
+    group = "kalilinux";
+    owner = "packages";
+    repo = "seclists";
+    rev = "upstream/${version}";
+    hash = "sha256-mJgCzp8iKzSWf4Tud5xDpnuY4aNJmnEo/hTcuGTaOWM=";
+  };
```

Comment:
We should get the source for the upstream location https://github.com/danielmiessler/SecLists. The repo in the Kali GitLab org seems to be a simple clone.

Reply:
Changes made!